docs/managing_deis/disk_usage.rst

```diff
@@ -0,0 +1,58 @@
+:title: Disk usage
+:description: Understanding disk usage for CoreOS and Deis.
+
+.. _disk_usage:
+
+Disk Usage
+==========
+
+When planning a Deis deployment, it's helpful to understand how both Deis and CoreOS utilize
+local storage on a machine.
+
+The following filesystem paths are the most important to consider:
+
+===================      =============================        ============================================================================================================================================
+location                 purpose                              considerations
+===================      =============================        ============================================================================================================================================
+/var/lib/etcd            etcd snapshot data                   etcd writes a relatively small amount of snapshot data here, so access should be as fast as possible (cloud providers use fast, local disks)
+/var/lib/docker          Docker image/volume storage          should be large - on cloud providers with external storage (AWS, GCE, Azure) this is a separate 100GB volume
+/var/lib/deis/store      mounted CephFS for deis-store        none - this is a virtually-mounted filesystem, and the "real" Ceph data lives in a Docker volume (so it's stored in /var/lib/docker)
```

Comment:
It's not a real filesystem - users don't manage this at all. That's exactly what the "consideration" comment is for.